casbin/model/model.cpp

```diff
@@ -54,7 +54,7 @@ bool Model :: HasSection(string sec) {
     return this->m.find(sec) != this->m.end();
 }
 
-void Model :: LoadSection(Model* model, ConfigInterface* cfg, string sec) {
+void Model :: LoadSection(Model* model, shared_ptr<ConfigInterface> cfg, string sec) {
```

Comment:
how about using `Model &` so no need to use a pointer.

Reply:
I don't think cfg is neccesery to be a shared_ptr,because we don't store the cfg as a member,we use it only one time.